pkg/cli/set/env.go

```diff
@@ -78,7 +78,7 @@ var (
 		oc set env -f dc.json ENV-
 
 		# Set some of the local shell environment into a deployment config on the server
-		oc set env | grep RAILS_ | oc env -e - dc/myapp
+		env | grep RAILS_ | oc env -e - dc/myapp
```

Comment:
Can we instead just drop this example, it doesn't gives us too much value but only clutters the overall help page for this command?

Reply:
@ammachado I'm open to get that also removed from `kubectl` as well.

Reply:
Example removed.